tests/integration/test_index.py

```diff
@@ -14,6 +14,72 @@
 
 from gqlalchemy.models import MemgraphIndex
 from gqlalchemy import Field, Node
+import pytest
+from gqlalchemy.exceptions import GQLAlchemyDatabaseMissingInNodeClassError
+
+
+def test_index_label(memgraph):
+    class User(Node, index=True, db=memgraph):
+        id: str
+
+    actual_index = memgraph.get_indexes()
+
+    assert set(actual_index) == {MemgraphIndex("User")}
+
+
+def test_missing_db_in_node_class(memgraph):
+    with pytest.raises(GQLAlchemyDatabaseMissingInNodeClassError):
+
```

Comment:
remove newline

Reply:
Black did this one and I think it has to stay to pass the tests

Reply:
You are right, my bad